docs/ide-usage.md

```diff
@@ -79,6 +79,10 @@ The following plugins are needed:
 
 1. To set up some static analysis tools, refer to [this document](static-analysis.md).
 
+1. To move on to the development phase, refer to [this document](development.md)
+
+
+
```

Comment:
Remove two blank lines here. The original file does not contain more than one blank line in a row.